plugins/modules/purefb_bucket.py

```diff
@@ -307,6 +319,25 @@ def create_bucket(module, blade):
                     module.params["name"]
                 )
             )
+        if MODE_VERSION in api_version:
+            if not module.params["block_new_public_policies"]:
+                module.params["block_new_public_policies"] = False
+            if not module.params["block_public_access"]:
+                module.params["block_public_access"] = False
+            pac = BucketPatch(
```

Comment:
Should this be `flashblade.BucketPatch`? BucketPatch is imported from `purity_fb` above which doesn't support `public_access_config`, resulting in errors like:
```
  File "/var/folders/jw/mbnzt6gs5637b_npdxx7m15c0000gn/T/ansible_purestorage.flashblade.purefb_bucket_payload_0c_j97nx/ansible_purestorage.flashblade.purefb_bucket_payload.zip/ansible_collections/purestorage/flashblade/plugins/modules/purefb_bucket.py", line 725, in <module>
  File "/var/folders/jw/mbnzt6gs5637b_npdxx7m15c0000gn/T/ansible_purestorage.flashblade.purefb_bucket_payload_0c_j97nx/ansible_purestorage.flashblade.purefb_bucket_payload.zip/ansible_collections/purestorage/flashblade/plugins/modules/purefb_bucket.py", line 707, in main
  File "/var/folders/jw/mbnzt6gs5637b_npdxx7m15c0000gn/T/ansible_purestorage.flashblade.purefb_bucket_payload_0c_j97nx/ansible_purestorage.flashblade.purefb_bucket_payload.zip/ansible_collections/purestorage/flashblade/plugins/modules/purefb_bucket.py", line 365, in create_bucket
**TypeError: BucketPatch.__init__() got an unexpected keyword argument 'public_access_config'**
```

Reply:
This appears to be an issue cause dby a subsequent patch. We will fix this and release a new Collection as this is an important function that is broken.

Reply:
Looking at this further, it appears a patch was that due to go into the 1.19.0 release wasn't merged correctly prior to the release.
This patch (#317) should provide the fix.
If you would like to check this out we will release 1.19.1 on your approval.

Reply:
Nice. Confirming that fixes the issue.